src/components/client/ClientNew.js

```diff
@@ -302,6 +347,19 @@ class NewClient extends React.Component {
                             className='fileInput'
                             accept='image/png, image/jpeg'
                           />
+                          <div className='validation-alert'>
+                            {this.state.showPopup
+                              ? this.state.errors.map((error, index) => {
+                                  return (
+                                    <Popup
+                                      error={error}
+                                      key={this.state.errors[index]}
+                                      onClose={this.togglePopupHandler}
+                                    />
+                                  );
+                                })
+                              : null}
+                          </div>
```

Comment:
why do we need the same code, div with pop up, twice in the code?

Reply:
One is when you upload logo and the other is when you change logo